packages/protocol/test/stability/exchange.ts

```diff
@@ -211,6 +211,64 @@ contract('Exchange', (accounts: string[]) => {
     })
   })
 
+  describe('#setSpread', () => {
```

Comment:
Sweet, thanks for the tests! Looks like `setStableToken` still needs one?

Reply:
whoops. Thanks!